inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClient.java

```diff
@@ -56,40 +58,57 @@
 import org.apache.inlong.manager.common.pojo.workflow.form.NewGroupProcessForm;
 import org.apache.inlong.manager.common.util.AssertUtils;
 import org.apache.inlong.manager.common.util.JsonUtils;
+import retrofit2.Call;
+import retrofit2.Retrofit;
+import retrofit2.converter.jackson.JacksonConverterFactory;
 
+import java.io.IOException;
 import java.util.List;
+import java.util.Map;
 
 /**
  * InnerInlongManagerClient is used to invoke http api of inlong manager.
  */
 @Slf4j
 public class InnerInlongManagerClient {
 
-    protected static final String HTTP_PATH = "api/inlong/manager";
-    private static final MediaType APPLICATION_JSON = MediaType.parse("application/json; charset=utf-8");
-    protected final OkHttpClient httpClient;
-    protected final String host;
-    protected final int port;
-    protected final String uname;
-    protected final String passwd;
     private final ObjectMapper objectMapper = new ObjectMapper();
 
+    private final InlongStreamApi inlongStreamApi;
+    private final InlongGroupApi inlongGroupApi;
+    private final StreamSourceApi streamSourceApi;
+    private final StreamTransformApi streamTransformApi;
+    private final StreamSinkApi streamSinkApi;
+    private final WorkflowApi workflowApi;
+
     public InnerInlongManagerClient(ClientConfiguration configuration) {
-        this.host = configuration.getBindHost();
```

Comment:
serviceUrl can be set as host1:port1,host2:port2,   so host, port is needed here

Reply:
my bad,I'll rollback this